staging/src/k8s.io/apiserver/pkg/registry/rest/delete.go

```diff
@@ -39,6 +39,7 @@ type GarbageCollectionPolicy string
 const (
 	DeleteDependents GarbageCollectionPolicy = "DeleteDependents"
 	OrphanDependents GarbageCollectionPolicy = "OrphanDependents"
+	Unsupported      GarbageCollectionPolicy = "Unsupported"
```

Comment:
> No. Not supporting GC means not supporting GC.

With this implementation, the `Unsupported` resource won't break the background deletion chain.